ui/src/views/Quickstart/index.jsx

```diff
@@ -135,15 +194,26 @@ const cmdDirectory = (type, org = '<YOUR_ORG>', repo = '<YOUR_REPO>') =>
   },
 }))
 export default class QuickStart extends Component {
-  state = {
-    ...initialState,
+  initialState = {
+    events: new Set([
+      'pull_request.opened',
+      'pull_request.reopened',
+      'pull_request.synchronize',
+    ]),
     owner: '',
     repo: '',
     access: 'collaborators',
     image: 'node',
     commands: cmdDirectory('node'),
     commandSelection: 'standard',
     installedState: null,
+    taskName: '',
+    taskDescription: '',
+  };
+
+  state = {
+    ...this.initialState,
+    condition: getMatchCondition(this.initialState.events),
   };
```

Comment:
This would make more sense if it was:

```jsx
state = this.initialState;
```

I think we can achieve that by doing something like:

```jsx
initialEvents = new Set([
  'pull_request.opened',
  'pull_request.reopened',
  'pull_request.synchronize',
]);

initialState = {
  events: this.initialEvents,
  condition: getMatchCondition(this.initialEvents),
  owner: '',
  repo: '',
  access: 'collaborators',
  image: 'node',
  commands: cmdDirectory('node'),
  commandSelection: 'standard',
  installedState: null,
  taskName: '',
  taskDescription: '',
};

state = this.initialState;
```